src/Pages/DonatePage.jsx

```diff
@@ -0,0 +1,312 @@
+import {
+  Accordion,
+  AccordionDetails,
+  AccordionSummary,
+  Box,
+  Container,
+  Grid,
+  makeStyles,
+  Typography,
+} from "@material-ui/core";
+import { ArrowDropDownOutlined } from "@material-ui/icons";
+import laptop from "../Assets/laptop.svg";
+import progress from "../Assets/progress.svg";
+import note from "../Assets/note_taking.svg";
+import discovery from "../Assets/science_discovery.svg";
+import code from "../Assets/qr_code.svg";
+
+const useStyles = makeStyles((theme) => ({
```

Comment:
We don't have to do this now, but something that may be a helpful refactor is to create separate style files. For example, we could create a file called `DonatePageStyles.ts` that has the `useStyles` object, that we can import and use in this file.

Reply:
Yes! I plan to do a pass on the theme and styles again once all the main pages are in place, I need to redo some of the fonts to be closer to what the hifis use that will also reduce makestyles code length, and there is time for moving the styles outside as well.